proto/viam/app/v1/billing.proto

```diff
@@ -62,20 +62,42 @@ message GetCurrentMonthUsageRequest {
   string org_id = 1;
 }
 
+enum UsageCostType {
+  USAGE_COST_TYPE_UNSPECIFIED = 0;
+  USAGE_COST_TYPE_DATA_UPLOAD = 1;
+  USAGE_COST_TYPE_DATA_EGRESS = 2;
+  USAGE_COST_TYPE_REMOTE_CONTROL = 3;
+  USAGE_COST_TYPE_STANDARD_COMPUTE = 4;
+  USAGE_COST_TYPE_CLOUD_STORAGE = 5;
+  USAGE_COST_TYPE_BINARY_DATA_CLOUD_STORAGE = 6;
+  USAGE_COST_TYPE_OTHER_CLOUD_STORAGE = 7;
+  USAGE_COST_TYPE_PER_MACHINE = 8;
+}
+
+message UsageCost {
+  UsageCostType resource_type = 1;
+  double cost = 2;
+}
+
+message ResourceUsageCosts {
+  repeated UsageCost usage_costs = 1;
+  double discount = 2;
+  double total_with_discount = 3;
+  double total_without_discount = 4;
+}
+
+message FragmentUsageCosts {
+  string fragment_id = 1;
+  ResourceUsageCosts resource_usage_costs = 2;
+}
+
 message GetCurrentMonthUsageResponse {
   google.protobuf.Timestamp start_date = 1;
   google.protobuf.Timestamp end_date = 2;
-  double cloud_storage_usage_cost = 3;
-  double data_upload_usage_cost = 4;
-  double data_egres_usage_cost = 5;
-  double remote_control_usage_cost = 6;
-  double standard_compute_usage_cost = 7;
-  double discount_amount = 8;
-  double total_usage_with_discount = 9;
-  double total_usage_without_discount = 10;
-  double per_machine_usage_cost = 11;
-  double binary_data_cloud_storage_usage_cost = 12;
-  double other_cloud_storage_usage_cost = 13;
+
+  ResourceUsageCosts org_usage = 14;
+  repeated FragmentUsageCosts fragment_usage_costs = 15;
```

Comment:
instead of making these so special cased with terminology such as "org" and "fragment", can we combine them? ultimately what we want to do here is support multiple billing tiers per org right? would be nice if this were an array of billing tiers + costs instead of two distinct separate variables and you could add a boolean like "defaultTier" to indicate which one is the org tier. picturing something like this (ignore my bad names) 

```
message BreakdownByBillingTiers {
  string tier_name = 1; 
  boolean is_default_tier = 2
  GroupedCosts ResourceUsageCosts = 3;
}

repeated BreakdownByBillingTiers breakdownByTier = 14;
```

i think this makes this API more flexible in the future. what do you think?

Reply:
great! looks much cleaner IMO. thanks!